TribalWars/src/ApplicationCore/Services/BuildingsQueueService.cs

```diff
@@ -43,25 +57,46 @@ public async Task<BuildingQueue> BuildingQueueByUserId(Guid userId)
             return await _buildingsQueueRepository.GetBuildingQueueByUserId(userId);
         }
 
-        public async Task AddBuildingsToQueue(int villageId,int buildingId)
+        public async Task<bool> CanAddToQueue(int villageId)
         {
             var buildingsInQueue = await _buildingsQueueRepository.GetQueueBuildings(villageId);
-            if(buildingsInQueue.Count < 2)
+            return buildingsInQueue.Count < 0;
```

Comment:
Did you change the approach of having 2 maximum buildings in queue? This code will always return false